functions/src/tests/utils.ts

```diff
@@ -87,22 +89,15 @@ export async function getOrRegisterIdToken(
 }
 
 /**
- * Retrieves the existing or registers a new API key for the given email.
- * @param email Email to register API key for.
- * @param idToken ID token to use for authentication.
- * @returns The API key.
+ * Inserts a mock API key document into the Firestore database.
```

Comment:
Is there something about this that makes it a "mock" key?  It'll be a real key, no?

Reply:
Good point--this isn't really a mock, I've removed that language 🙏